src/sources/soundsourceffmpeg.cpp

```diff
@@ -20,30 +22,47 @@ constexpr uint64_t kavChannelLayoutUndefined = 0;
 
 constexpr int64_t kavStreamDefaultStartTime = 0;
 
-// Use 0-based sample frame indexing
-constexpr SINT kMinFrameIndex = 0;
+// https://ffmpeg.org/doxygen/trunk/structAVPacket.html#details
+// "For audio it may contain several compressed frames."
+// A stream packet may produce multiple stream frames when decoded.
```

Comment:
No idea and not documented. What shall I do?

After reading a packet we need to consume all decoded frames, because the raw packet is discarded after leaving the decoding loop. This constant only prevents reallocation of the internal buffer. A warning log message indicates, if a reallocation occurs.